README.md

```diff
@@ -6,7 +6,22 @@ alp is Access Log Profiler
 
 # Installation
 
-Download from https://github.com/tkuchiki/alp/releases
+### Binary distribution
+
+You can just pick your download [here](https://github.com/tkuchiki/alp/releases), put it in your PATH as alp, and set it as executable, e.g.
+
+```bash
+sudo cp <downloaded file> /usr/local/bin/alp
+sudo chmod a+x /usr/local/bin/alp
```

Comment:
I think that it is better for the following command:
Could you let me know what you think about it?

```diff
- sudo cp <downloaded file> /usr/local/bin/alp
- sudo chmod a+x /usr/local/bin/alp
+ sudo install <downloaded file> /usr/local/bin/alp
```

Reply:
It makes sense.